src/pinball_wizard.coffee

```diff
@@ -135,8 +135,9 @@ define ->
     JSON.parse(storage.getItem('pinball_wizard') or "[]") or []
 
   activatePermanently = (name) ->
-    appendPermanent(name)
-    activate(name, 'permanent')
+    for n in name.split(',')
```

Comment:
What do you think about using coffeescript splat? http://coffeescript.org/#splats